github_test.go

```diff
@@ -100,58 +170,38 @@ var _ = Describe("GitHub Client", func() {
 		})
 	})
 
-	Describe("when the source is configured with the deprecated user field", func() {
-		BeforeEach(func() {
-			source = Source{
-				User:       "some-owner",
-				Repository: "some-repo",
-			}
-
-			server.AppendHandlers(
-				ghttp.CombineHandlers(
-					ghttp.VerifyRequest("GET", "/repos/some-owner/some-repo/releases"),
-					ghttp.RespondWith(200, "[]"),
-				),
-			)
-		})
-
-		It("uses the provided user as the owner", func() {
-			_, err := client.ListReleases()
-			Ω(err).ShouldNot(HaveOccurred())
-		})
-	})
-
 	Describe("ListReleases", func() {
 		BeforeEach(func() {
 			source = Source{
-				Owner:      "concourse",
-				Repository: "concourse",
+				Owner:       "concourse",
+				Repository:  "concourse",
+				AccessToken: "test",
 			}
 		})
-		Context("When list of releases return more then 100 items", func() {
```

Comment:
Could you add back the tests for the v3 API as well (for unauthenticated requests), now that we still use the v3 API when there's no access token